source/views/dictionary/detail.js

```diff
@@ -2,9 +2,14 @@
 import React from 'react'
 import {StyleSheet} from 'react-native'
 import {Markdown} from '../components/markdown'
+import {ListFooter} from '../components/list'
 import glamorous from 'glamorous-native'
 import type {WordType} from './types'
 
+// TODO: This doesn't point at the SA dictionary because they don't have an
```

Comment:
If you wanted to get tricky, you could analyze the first letter of the word and determine where it belonged between [a-g](https://wp.stolaf.edu/sa/a-g/), [h-o](https://wp.stolaf.edu/sa/h-o/), and [p-z](https://wp.stolaf.edu/sa/p-z/). Otherwise, you could just link to the start page of [a-g](https://wp.stolaf.edu/sa/a-g/).

Reply:
I don't want to do that; I'm planning to ask and make a landing page, so these pages are liable to move soon.

Reply:
Okay.